packages/mendel-generator-prune/README.md

```diff
@@ -0,0 +1,30 @@
+# mendel-generator-prune
+
+Mendel-generator modifies references of exposed modules across bundles.
+
+In a large application with lot of files, few common files are required by most of other files in other bundles. This generator assigns unique hash to each exposed module. Instead of referring those common files by file path, they will be referred by unique hash. Hash is computed using `shasum`.
```

Comment:
This is not exactly what it does.

Before:
1) It did remove dangling files, such as files required initially and later not used because of some transformation (examples would be `requires` removed after `envify` and some CSS-in-JS dependency.
2) Make sure we replace `normalizedId` (which is relative to variation and is already unique) by a counter, so that the files become really smaller.

After `shasum`: 
1) Still the same as above
2) Replace all file names by something else that still ensures cross bundle consistency. (same thing normalizedId already does?)

Now that I I am thinking here. Have you guys tried just not using `shasum` and not replacing by any numbers? just `normalizedId` and nothing else? Might actually solve the issue. If we remove the part of the code that renames the entries/entrances we can just keep reason 1 explained on the readme. To be honest, I think `normalizedIds` will compress better. Drawback is: they are readable by user that view source.